crates/blockifier/src/execution/execution_utils.rs

```diff
@@ -124,17 +124,19 @@ pub fn execute_entry_point_call(
     state: &mut dyn State,
     context: &mut EntryPointExecutionContext,
 ) -> EntryPointExecutionResult<CallInfo> {
-    match compiled_class {
+    let pre_time = std::time::Instant::now();
+
+    let mut result = match compiled_class {
```

Comment:
Don't really know how much overhead does the match could imply, but perhaps it would be better the define a variable and measure the time inside the match. Just to make sure that what we are measuring  it the entrypoint call.

Reply:
Done